src/Arcus.EventGrid.Tests.Unit/Parsing/EventGridParsingTests.cs

```diff
@@ -36,5 +37,100 @@ public void Parse_ValidSubscriptionValidationEvent_ShouldSucceed()
             Assert.NotNull(eventPayload.Data);
             Assert.Equal(validationCode, eventPayload.Data.ValidationCode);
         }
+
+        [Fact]
+        public void Parse_ValidSubscriptionValidationEventWithSessionId_ShouldSucceed()
+        {
+            // Arrange
+            string rawEvent = EventSamples.SubscriptionValidationEvent;
```

Comment:
Just curious: what are the guidelines regarding the usage of var?

Reply:
See https://github.com/arcus-azure/arcus.eventgrid/pull/36#issuecomment-432599967

WIP but will be enforced in the future.